akka-typed/src/test/scala/docs/akka/typed/IntroSpec2.scala

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (C) 2014-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package docs.akka.typed
+
+//#imports
+import akka.typed._
+import akka.typed.scaladsl.Actor._
+import akka.typed.AskPattern._
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.concurrent.Await
+//#imports
+import akka.testkit.AkkaSpec
+import akka.typed.TypedSpec
+
+object IntroSpec2 {
```

Comment:
@rkuhn To get a feeling for it I have tried to convert the IntroSpec. Some questions, see TODOs here...